backend/serdes/utils.py

```
@@ -77,7 +77,7 @@
     AppliedControl,
     Evidence,
     Framework,
-    Project,
+    Perimeter,
```

Ruff (0.8.2)

Redefinition of unused `Perimeter` from line 19

Remove definition: `Perimeter`

(F811)